neurolib/optimize/evolution/evolution.py

```diff
@@ -414,8 +455,9 @@ def runInitial(self):
         # save all simulation data to pypet
         self.pop = eu.saveToPypet(self.traj, self.pop, self.gIdx)
 
-        # Only the best indviduals are selected for the population the others do not survive
-        self.pop[:] = self.toolbox.selBest(self.pop, k=self.traj.popsize)
+        # reduce initial population to popsize
+        #self.pop = self.toolbox.select(self.pop, k=self.traj.popsize, **self.SELECT_P)
+
```

Comment:
We do not need to reduce the initial population here because it will be done in the first evolution round anyway....